plugins/tiff/src/tiffpresentation.cc

```diff
@@ -422,14 +431,18 @@ PipetteLayerOperations::PipetteColor operator/(PipetteLayerOperations::PipetteCo
 ////////////////////////////////////////////////////////////////////////
 // PipetteViewInterface
 ////////////////////////////////////////////////////////////////////////
-PipetteLayerOperations::PipetteColor TiffPresentation::getAverages(Scroom::Utils::Rectangle<int> area)
+PipetteLayerOperations::PipetteColor TiffPresentation::getPixelAverages(Scroom::Utils::Rectangle<int> area)
 {
-  PipetteLayerOperations::Ptr pipetteLayerOperation = boost::dynamic_pointer_cast<PipetteLayerOperations>(this->ls[0]);
+  PipetteLayerOperations::Ptr pipetteLayerOperation = this->pipetteLayer;
```

Comment:
Now replace `pipetteLayerOperation` with `pipetteLayer` (no `this->`) everywhere :wink: